package.json

```diff
@@ -3,7 +3,8 @@
   "version": "1.0.0",
   "private": true,
   "scripts": {
-    "start": "node ./bin/www"
+    "start": "node ./bin/www",
+    "test": "NODE_ENV=test mocha test"
```

Comment:
@juancarlospaco because you introduced the _dotenv-safe_ dependency there is no step in where the user exports the env variables.

You have to explicitly set the env variables for testing purposes here. Like this:

```
"test": "NODE_ENV=test TWILIO_ACCOUNT_SID=AC2XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX TWILIO_AUTH_TOKEN=XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX TWILIO_NUMBER=+15551230987 MONGO_URL_TEST=mongodb://localhost/appointment-reminders-node-test ./node_modules/mocha/bin/mocha test"
```